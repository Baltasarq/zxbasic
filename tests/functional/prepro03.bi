
#define test x 'a comment \
#define test2 y ' This define should not be ignored

test
test2




