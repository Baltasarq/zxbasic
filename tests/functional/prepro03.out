#line 1 "prepro03.bi"




x 
y 




